utils/playbook.py

```diff
@@ -18,7 +18,7 @@ def run_playbook(provider: str, username: str, project_name: str, curr_working_d
 
     with open(log_file, 'a+'):
         try:
-            runner = run_async(playbook=playbook_path, inventory=inventory, envvars=env_vars, extravars=extra_vars, quiet=True)
+            runner = run_async(playbook=playbook_path, inventory=inventory, envvars=env_vars, extravars=extra_vars, quiet=False)
```

Comment:
why are we using run_playbook in aws while we are using run_async in azure and gcp???